mesh/mesh_test.go

```diff
@@ -395,14 +395,15 @@ func TestMesh_AddBlockWithTxs_PushTransactions_getInvalidBlocksByHare(t *testing
 	var blocks []*types.Block
 	blocks = append(blocks, addBlockWithTxs(r, msh, layerID, true, tx1, tx2))
 	blocks = append(blocks, addBlockWithTxs(r, msh, layerID, true, tx2, tx3, tx4))
-	blocks = append(blocks, addBlockWithTxs(r, msh, layerID, true, tx4, tx5))
+	blocks = append(blocks, addBlockWithTxs(r, msh, layerID, true, tx4))
 	blocks = append(blocks, addBlockWithTxs(r, msh, layerID, true, tx5))
 	hareBlocks := blocks[:3]
 
 	invalid := msh.getInvalidBlocksByHare(types.NewExistingLayer(layerID, hareBlocks))
 	r.ElementsMatch(blocks[3:], invalid)
```

Comment:
The original test also checked that a transaction that appear in both valid and invalid blocks are treated as valid. I don't think it's super critical, but fixing this is easy (in the original test there were two valid and one invalid blocks):
```suggestion
	blocks = append(blocks, addBlockWithTxs(r, msh, layerID, true, tx4, tx5))
	hareBlocks := blocks[:2]

	invalid := msh.getInvalidBlocksByHare(types.NewExistingLayer(layerID, hareBlocks))
	r.ElementsMatch(blocks[2:], invalid)
```

(I've run this modification locally to ensure it works, so it should be safe to commit my suggestion)